### PR TITLE
Add stored NFT data Bubblegum section

### PR DESCRIPTION
**Pull request by danenbm**
Link to preview of specific page I updated: https://developer-hub-git-danenbm-stored-nft-metaplex.vercel.app/bubblegum/stored-nft-data

### CODE REVIEW

**Comment by MarkSackerberg** [comment]
Looking good to me.
Shouldn't we link the different reference implementations you are explaining in the list though? Either to their docs (which there are none currently i assume) or their github repositories?

File: src/pages/bubblegum/stored-nft-data.md [hunk 1]
```diff
@@ -4,4 +4,72 @@ metaTitle: Bubblegum - Storing and Indexing NFT Data
 description: Learn more about how NFT data is stored on Bubblegum
 ---
 
-_Coming soon..._
+As mentioned in the [Overview](/bubblegum#read-api), whenever compressed NFTs (cNFTs) are created or modified, the corresponding transactions are recorded on-chain in the ledger, but the cNFT state data is not stored in account space.  This is the reason for the massive cost savings of cNFTs, but for convenience and usability, the cNFT state data is indexed by RPC providers and available via the **the Read API**.
```
Comment (febo) [inline src/pages/bubblegum/stored-nft-data.md:7]: Should this say "Metaplex Read API" and link to the public repo?

**Review by blockiosaurus — APPROVED** [review]

File: src/pages/bubblegum/stored-nft-data.md (hunk 1 above)
Comment (danenbm) [reply src/pages/bubblegum/stored-nft-data.md]: Renamed it to "Metaplex Read API" here and where it is introduced in the Overview page (index.md).

**Comment by danenbm** [comment]
> Looking good to me. Shouldn't we link the different reference implementations you are explaining in the list though? Either to their docs (which there are none currently i assume) or their github repositories?

I wrote some additional text explaining how different RPC providers use some or all of our reference code, and added the link to our Metaplex reference implementation.

I also added a link to the RPC provider page, which shows the different RPCs who support the Metaplex standard, and has links to their detailed API documentation.  I think since they do have docs we want to be linking to those, rather than to github repos.

Let me know what you think.  Thanks!

**Review by febo — APPROVED** [review]
LGTM

**Pull request merged**